example/extending/newlang/5-hello-python/build.mill

```diff
@@ -0,0 +1,21 @@
+package build
+import mill._
+
+def installPython = Task{
```

Comment:
I wonder if it would be better to just test that python is installed and if not, let the user deal with it since this solution here is architecture dependent. It wouldn't work for any macos or win users.

Reply:
Alright.. noted

I was even thinking about finding way to install for the three main OSes but ran out of luck. Yours is better